install/uninstall.go

```diff
@@ -141,3 +145,29 @@ func (k *K8sUninstaller) waitForPodsToBeDeleted(ctx context.Context) error {
 		}
 	}
 }
+
+func (k *K8sUninstaller) enableCNIUninstall(ctx context.Context) {
+	pods, err := k.client.ListPods(ctx, k.params.Namespace, metav1.ListOptions{LabelSelector: defaults.AgentPodSelector})
+	if err != nil {
+		k.Log("❌ Failed to enable cni cleanup: %v", err)
+		return
+	}
+	wp := workerpool.New(k.params.WorkerCount)
+
+	for i := range pods.Items {
+		pod := &(pods.Items[i]) // go iterator silliness; can't use range variable
```

Comment:
How about something like
```suggestion
	for _, pod := range pods.Items {
		pod := pod // capture range variable
```

Reply:
oh right, that's the trick for that.